### PR TITLE
Schemas ver1

### PR DESCRIPTION
**Pull request by edsu7**

### CODE REVIEW

File: schemas/participant.json [hunk 1]
```diff
@@ -0,0 +1,137 @@
+{
+    "name": "participant",
+    "description": "The collection of data elements related to a specific donor in an ARGO program.",
```
Comment (lindaxiang) [inline schemas/participant.json:3]: ARGO --> PCGL

File: schemas/sample.json [hunk 2]
```diff
@@ -1,21 +1,84 @@
 {
-  "name": "sampe",
-  "description": "A description",
+  "name": "sample",
+  "description": "The collection of data elements required to register the required Donor-Specimen-Sample data to the ARGO Data Platform. Registration of samples is required before molecular and clinical data submission can proceed.",
```
Comment (lindaxiang) [inline schemas/sample.json:3]: ARGO --> PCGL

**Review by lindaxiang — CHANGES_REQUESTED** [review]

File: schemas/participant.json [hunk 3]
```diff
@@ -0,0 +1,137 @@
+{
+    "name": "participant",
+    "description": "The collection of data elements related to a specific donor in a PCGL program.",
```
Comment (guanqiaofeng) [inline schemas/participant.json:3]: addressing:
https://github.com/Pan-Canadian-Genome-Library/data-dictionary/pull/1#discussion_r1534306519

File: schemas/sample.json [hunk 4]
```diff
@@ -1,21 +1,84 @@
 {
-  "name": "sampe",
-  "description": "A description",
+  "name": "sample",
+  "description": "The collection of data elements required to register the required Donor-Specimen-Sample data to the PCGL Data Platform. Registration of samples is required before molecular and clinical data submission can proceed.",
```
Comment (guanqiaofeng) [inline schemas/sample.json:3]: addressing https://github.com/Pan-Canadian-Genome-Library/data-dictionary/pull/1#discussion_r1534308416

**Review by lindaxiang — APPROVED** [review]

**Review by leoraba — CHANGES_REQUESTED** [review]

File: schemas/participant.json [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+                "required": true,
+                "regex": "#/regex/submitter_id"
+            }
+        },
+        {
+            "name": "sex_at_birth",
+            "description": "Refers to sex assigned at birth. Sex at birth is typically assigned based on a person's reproductive system and other physical characteristics. The provided values are based on the categoried defined by Statistics Canada",
+            "valueType": "string",
+            "meta": {
+                "core": true,
```
Comment (joneubank) [inline schemas/participant.json:43]: Do we have a definition and meaning for what "core" field means? This feels like a carry over from ARGO that doesn't have a PCGL equivalent.

File: schemas/participant.json [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+            }
+        },
+        {
+            "name": "date_of_birth",
+            "description": "Indicate participant's date of birth.",
+            "valueType": "string",
+            "meta": {
+                "core": true,
+                "displayName": "Date Of Birth",
+                "notes": "Source: MOH; All4One : Month,year of birth; mCODE STU3: Patient.birthDate; Phenopacket: Individual.date_of_birth; Beacon V2: Individual.date_of_birth"
```
Comment (joneubank) [inline schemas/participant.json:115]: We put `Source` in all our notes... would we want to just add a `source` property to our meta?

Similarly, would we want a separate property to list the field from other data sets that maps here?
ex. `"mapping": "All4One:Month,year of birth|mCODE STU3: Patient.birthDate|Phenopacket:Individual.date_of_birth"` etc.

File: references/regex.json [hunk 7]
```diff
@@ -1 +1,5 @@
-{}
+{
+    "submitter_id": "^[A-Za-z0-9\\-\\._]{1,64}$",
```
Comment (joneubank) [inline references/regex.json:2]: Do we really need to restrict submitters to a max length 64 restriction? I think our only rule is the need for min length of 1, not sure we care about max length at all.

```suggestion
    "submitter_id": "^[A-Za-z0-9\\-\\._]+$",
```

File: references/regex.json [hunk 8]
```diff
@@ -1 +1,5 @@
-{}
+{
+    "submitter_id": "^[A-Za-z0-9\\-\\._]+$",
```
Comment (edsu7) [inline references/regex.json:2]: As per jon's suggestion:
https://github.com/Pan-Canadian-Genome-Library/data-dictionary/pull/1#discussion_r1541284510

File: references/regex.json [hunk 9]
```diff
@@ -1 +1,5 @@
...
+    "yyyy_mm": "^\\d{4}-(0[1-9]|1[012])$",
```
Comment (edsu7) [inline references/regex.json:3]: As per Jon's suggestion:
https://github.com/Pan-Canadian-Genome-Library/data-dictionary/pull/1#discussion_r1541278221

File: schemas/participant.json [hunk 10]
```diff
@@ -0,0 +1,241 @@
...
+      },
+      {
+        "name": "submitter_participant_id",
+        "valueType": "string",
+        "description": "Unique identifier for the participant, assigned by the data provider.",
+        "meta": {
+          "primaryId": true,
+          "examples": "90234,BLD_donor_89,AML-90",
+          "displayName": "Submitter Participant ID",
+          "source": "MOH/ARGO",
```
Comment (edsu7) [inline schemas/participant.json:32]: As per jon's suggestion:
https://github.com/Pan-Canadian-Genome-Library/data-dictionary/pull/1#discussion_r1541271035

Source and mappings were added to meta

File: schemas/participant.json [hunk 11]
```diff
@@ -0,0 +1,241 @@
...
+          ],
+          "notes": null
+        },
+        "restrictions": {
+          "required": true,
+          "regex": "#/regex/submitter_id"
+        }
+      },
+      {
+        "name": "sex_at_birth",
```
Comment (edsu7) [inline schemas/participant.json:59]: `core` has been removed as per:
https://github.com/Pan-Canadian-Genome-Library/data-dictionary/pull/1#discussion_r1541264488

File: schemas/participant.json [hunk 12]
```diff
@@ -0,0 +1,241 @@
...
+        "description": "Refers to sex assigned at birth. Sex at birth is typically assigned based on a person's reproductive system and other physical characteristics. The provided values are based on the categoried defined by Statistics Canada",
+        "valueType": "string",
+        "meta": {
+          "displayName": "Sex At Birth",
+          "notes": "EDI indicator",
```
Comment (edsu7) [inline schemas/participant.json:64]: @lindaxiang do we want EDI indicator as its own field?

File: schemas/index.js [hunk 13]
```diff
@@ -1,4 +1,7 @@
 // THIS IS A SAMPLE SCHEMA
+import study from "./study.json" assert { type: "json" };
+import participant from "./participant.json" assert { type: "json" };
 import sample from "./sample.json" assert { type: "json" };
+import specimen from "./specimen.json" assert { type: "json" };
 
-export default { sample };
+export default { study, participant, sample, specimen };
```
Comment (leoraba) [inline schemas/index.js:7]:
```suggestion
export default [study, participant, sample, specimen];
```

**Review by leoraba — CHANGES_REQUESTED** [review]
change default export as an array

**Comment by edsu7** [comment]
Dictionary in lectern:
https://lectern.overture-sandbox.cumulus.genomeinformatics.org/dictionaries/660c583ec7e3a638393ac462

File: schemas/participant.json [hunk 14]
```diff
@@ -0,0 +1,142 @@
...
+          "source": "MOH"
+        },
+        "restrictions": {
+          "codeList": "#/list/sex_at_birth",
+          "required": true
+        }
+      },
+      {
+        "name": "gender",
+        "description": "Refers to an individual's personal and social identity as a man, woman or non-binary person (a person who is not exclusively a man or a woman). The provided values are based on the categoried defined by Statistics Canada",
```
Comment (lindaxiang) [inline schemas/participant.json:57]: little typo: categoried --> categories

**Review by lindaxiang — APPROVED** [review]

**Pull request merged**